mimic_alpha.py

```diff
@@ -189,6 +191,81 @@ def colorAlpha_to_rgb(colors, alpha, bg='w'):
 
     return rgb
 
+
+def cmap(cmap_name, alpha, bg="w"):
+    """
+    Generate an RGB colormap from a given mpl cmap and alpha value.
+
+    Parameters:
+    -----------
+    cmap_name: String
+       A standard Matplotlib colormap name:
+       http://matplotlib.org/examples/color/colormaps_reference.html
+    alpha: Float
+       Value of alpha to mimic in range [0,1].
+    bg: Matplotlib color
+       Color of the background.
+
```

Comment:
Add the return value, please:

    Returns
    ----------
    ma_cmap : :class:`matplotlib.colors.Colormap`
         add your explanation